apps/prairielearn/elements/pl-drawing/mechanicsObjects.js

```diff
@@ -2520,33 +2514,29 @@ mechanicsObjects.byType['pl-axes'] = class extends PLDrawingBaseElement {
     obj.selectable = false;
 
     // Adding x-axis
-    var options_axis_1 = _.defaults(
-      {
-        left: options.left - options.xneg,
-        top: options.top,
-        width: options.xneg + options.xpos,
-        drawEndArrow: true,
-        arrowheadWidthRatio: 1.5,
-        arrowheadOffsetRatio: 1.5,
-      },
-      options,
-    );
+    var options_axis_1 = {
+      ...options,
+      left: options.left - options.xneg,
+      top: options.top,
```

Comment:
I believe this line can be removed.

Reply:
All other objects around this one have top and left explicitly set. This was also not needed in the previous version, but I believe keeping it there makes it clear that the top of the x-axis matches the top of the overall object. What do you think?

Reply:
I don't feel strongly one way or the other, I'm happy to leave it in if you prefer.